### ISSUE TITLE
Windows 10 Platform Error while handling server request. This path could not be converted to a browser path.

### ISSUE THREAD

**Issue by Kreminskyi** [SVG-Edit/svgedit#518]
Microsoft Windows [Version 10.0.19041.985]
(c) Microsoft Corporation. All rights reserved.

C:\WINDOWS\system32>cd c:\xampp\htdocs\svgtest\

c:\xampp\htdocs\svgtest>npm i
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!

> core-js@2.6.11 postinstall c:\xampp\htdocs\svgtest\node_modules\@babel\polyfill\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

> core-js@2.6.11 postinstall c:\xampp\htdocs\svgtest\node_modules\babel-register\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

> core-js@2.6.11 postinstall c:\xampp\htdocs\svgtest\node_modules\babel-runtime\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

> core-js@3.12.1 postinstall c:\xampp\htdocs\svgtest\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

> core-js-pure@3.9.0 postinstall c:\xampp\htdocs\svgtest\node_modules\core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"

> core-js-bundle@3.12.1 postinstall c:\xampp\htdocs\svgtest\node_modules\core-js-bundle
> node -e "try{require('./postinstall')}catch(e){}"

> cypress@7.3.0 postinstall c:\xampp\htdocs\svgtest\node_modules\cypress
> node index.js --exec install

Cypress 7.3.0 is installed in C:\Users\user\AppData\Local\Cypress\Cache\7.3.0

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\unified-args\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\rollup\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\mocha\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules\@web\dev-server-core\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"ia32"})

added 2247 packages from 935 contributors and audited 2263 packages in 61.641s

170 packages are looking for funding
  run `npm fund` for details

found 39 vulnerabilities (11 low, 15 moderate, 13 high)
  run `npm audit fix` to fix them, or `npm audit` for details

c:\xampp\htdocs\svgtest>npm run start

> svgedit@7.0.0-preview start c:\xampp\htdocs\svgtest
> web-dev-server --node-resolve

Web Dev Server started...

  Root dir: c:\xampp\htdocs\svgtest
  Local:    http://localhost:8000/
  Network:  http://192.168.56.1:8000/

**Error while handling server request.
PluginError:

Resolved ./jquery.min.js to C:\xampp\htdocs\svgtest\src\editor\jquery.min.js.

This path could not be converted to a browser path. Please file an issue with a reproduction.**
    at Object.resolveImport (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-rollup\dist\rollupAdapter.js:161:35)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Object.resolve (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-rollup\dist\createRollupPluginContextAdapter.js:52:36)
    at async Object.resolveImport (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-rollup\dist\rollupAdapter.js:100:30)
    at async resolveImport (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-core\dist\plugins\transformModuleImportsPlugin.js:127:30)
    at async transformImport (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-core\dist\plugins\transformModuleImportsPlugin.js:140:37)
    at async maybeResolveImport (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-core\dist\plugins\transformModuleImportsPlugin.js:68:41)
    at async transformImports (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-core\dist\plugins\transformModuleImportsPlugin.js:94:36)
    at async Object.transform (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-core\dist\plugins\transformModuleImportsPlugin.js:171:49)
    at async c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-core\dist\middleware\pluginTransformMiddleware.js:44:32
Terminate batch job (Y/N)? n

c:\xampp\htdocs\svgtest>node -v
v14.17.0

c:\xampp\htdocs\svgtest>npm -v
6.14.13

GET http://localhost:8000/src/editor/index.js net::ERR_ABORTED 500 (Internal Server Error)

**Comment by jfhenon** [comment]
Hello @Kreminskyi ,
I'm not sure it's the cause of your error but as your message indicates an issue with the jquery library, I merged a PR that is stopping to use the old local jquery file to use the latest npm version. 
Hopefully, this will fix your issue. 
Can you please update to the latest master and try your test again? 
Thanks
JF

**Comment by Kreminskyi** [comment]
Thanks @jfhenon 
The next one appeared:

Web Dev Server started...

  Root dir: c:\xampp\htdocs\svgtest
  Local:    http://localhost:8000/
  Network:  http://192.168.56.1:8000/

Error while handling server request.
PluginError:

Resolved ./Editor.js to C:\xampp\htdocs\svgtest\src\editor\Editor.js.

This path could not be converted to a browser path. Please file an issue with a reproduction.
    at Object.resolveImport (c:\xampp\htdocs\svgtest\node_modules\@web\dev-server-rollup\dist\rollupAdapter.js:161:35)

**Comment by jfhenon** [comment]
Would you please run `npm run build` and then launch `http://localhost:8000/dist/editor/index.html`

**Comment by Kreminskyi** [comment]
Hello! @jfhenon 
After running "build"
src/editor/index.js → dist/editor/index.js, dist/editor/xdomain-index.js...
(node:6640) ExperimentalWarning: The fs.promises API is experimental
recreating dist

**Comment by Kreminskyi** [comment]
Hello!
@jfhenon 
Here is result in Chrome :
locale/lang.en.js:1 Failed to load resource: the server responded with a status of 404 (Not Found)
jquery.slim.js:523 Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: http://localhost:8000/dist/editor/locale/lang.en.js

**Comment by jfhenon** [comment]
Do you have a recent version of Chrome?

**Comment by Kreminskyi** [comment]
Hello!
Yes, the version is 90.0.4430.212.

**Comment by Kreminskyi** [comment]
How about this : @jfhenon 
Error while handling server request.
PluginError:

Resolved ./Editor.js to C:\xampp\htdocs\svgtest\src\editor\Editor.js.

**Comment by jfhenon** [comment]
I'm struggling on your issue. It looks like an issue with rollup or web-dev-server but I don't have a Win10 environment to reproduce it.

**Comment by ValiantHalibut** [comment]
I was having the same issue, turns out that it was present in web-dev-server - see here: [https://github.com/modernweb-dev/web/issues/1483](https://github.com/modernweb-dev/web/issues/1483)

Short version - Windows paths are not case sensitive but the resolved paths in web-dev-server _are,_ so when the resolver attempts to provide absolute paths to the node modules in order to generate browser paths it is unable to find the referenced imports.

I came across this when running web-dev-server from Powershell after navigating to a project folder using terminal commands that had inconsistent casing. So if my project folder lived here:
`C:\Dev\Projects\MyProject`
and I navigated to that folder by typing 
`cd c:\dev\projects\myproject`
and then attempted to run web-dev-server, I would get that error. Resolve it by simply using the Set-Location cmdlet with a case sensitive path: `cd C:\Dev\Projects\MyProject`
